src/components/NFTEditionSelect.vue

```diff
@@ -1,128 +1,58 @@
 <template>
-  <div :class="rootClasses">
-    <table
-      v-if="(!isSingleItem || (isSingleItem && items[0].name)) && !isAllSoldOut"
-      class="border-separate border-spacing-y-[8px] mb-[8px] w-full"
-    >
-      <tbody>
-        <NFTEditionSelectItem
-          v-for="(item, index) in items"
-          :key="index"
-          :name="item.name"
-          :currency="item.currency"
-          :price-label="item.priceLabel"
-          :stock="item.stock"
-          :is-selected="item.value === selectedValue"
-          :current-price="item.price"
-          :default-price="item.defaultPrice"
-          @click="handleClickPriceSelectItem(item)"
-        />
-      </tbody>
-    </table>
-
+  <div class="flex items-center gap-[12px] w-full flex-wrap">
     <div
-      :class="[
-        'flex',
-        'flex-wrap',
-        'justify-end',
-        'items-stretch sm:items-center',
-        'gap-[12px]',
-        'flex-col sm:flex-row',
-      ]"
-    >
-      <ButtonV2
-        v-if="!isAllSoldOut"
-        preset="plain"
-        class="text-white underline max-638:order-1"
-        :text="
-          $t(
-            isSingleItem
-              ? 'nft_edition_view_edition_button_text'
-              : 'nft_edition_select_compare_button_text'
-          )
-        "
-        @click="handleClickCompareItemsButton"
-      />
-      <template v-if="isSingleItem || isAllSoldOut">
-        <span v-if="isSingleItem && !items[0].name" class="text-white">{{
-          priceLabel
-        }}</span>
-        <NFTStockLabel
-          v-if="items.length > 1 || isAllSoldOut"
-          :stock="stock"
-          :is-dark="!isAllSoldOut"
-        />
-      </template>
-
+      class="hidden laptop:block border-b-[1px] border-[#ebebebeb] w-full my-[16px]"
+    />
+    <div class="text-[18px] font-500 text-dark-gray w-full">
+      {{ $t('nft_edition_select_section_label') }}
+    </div>
```

Comment:
use `v-text` for content

Reply:
[💄 Add discount original price label](https://github.com/likecoin/liker-land/pull/1948/commits/e8fdc2a56a0301fbd7904ec5cdb9f9cdd33c73a7)